src/entry.js

```diff
@@ -46,17 +50,10 @@ module.exports = argv => {
     process.exit(1);
   }
 
-  const notifier = updateNotifier({ pkg: require('../package.json') });
-
-  if (notifier.update) {
-    notifier.notify({
-      message: `Update available ${colors.grey(
-        notifier.update.current
-      )} → ${colors.green(notifier.update.latest)}\nRun ${colors.cyan(
-        'npm i -g zapier-platform-cli'
-      )} to update, and then ${colors.blue('re-test your integration')}.`
-    });
-  }
+  updateNotifier({
+    pkg: require('../package.json'),
+    updateCheckInterval: UPDATE_NOTIFICATION_INTERVAL
```

Comment:
The reason why we added this specific message was to make it clearer what needed to be done by the developer (highlighting the problem, command, and remind to re-test).

It was part of a UX research/review, so I'm not so sure we'd want to remove it.

If we found out something more about this in the meanwhile, then it's perfectly fine to change! #recommendation

Reply:
I like the clear instructions element of it, but  I felt compelled to change it because the message was misleading. It said to "run `npm i -g zapier-platform-cli` and then re-test your integration". Since the test code will break when `core` is updated (not cli), re-running `test` there doesn't reflect the new version. A better message might describe updating the dep in `package.json` and then re-running npm install, but that on top of wanting them to update CLI seemed to get really wordy for a quick "pop up". Also, assuming that they're not updating across a major version, there's really no need to re-test. The semver package allows us to get major|minor|patch from a pair of versions (`.difference`) so we could selectively include the re-test message if it's important. 

Was the concern on the original UX finding that the dev wouldn't know how to update the dependency or wasn't sure what effect doing so would have? I feel like I'm missing some context there. Do you know offhand where the result of those test(s) were?

Reply:
@xavdid has a point. Maybe we should move the message ("re-test your integration") to where we notify upgrading `core`?

Reply:
That was a request by @benzapier and @ZaneLyon IIRC. I didn't check where the data for that request came from, but seemed anecdotal.

I agree the message would be quite large for the whole correct instruction, so maybe @eliangcs 's suggestion makes more sense.

Reply:
Sweet, grabbed the best of both worlds. 

![](https://cdn.zapier.com/storage/photos/11a7add1b24e47e9b0948093f9e7e8a5.png)

Reply:
er, might swap the version numbers to after the filename

Reply:
This is the version i'm pushing:

![](https://cdn.zapier.com/storage/photos/413d06a77147476ea84de60880c6cfaf.png)

Reply:
Thanks, that looks awesome!